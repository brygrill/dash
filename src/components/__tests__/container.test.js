import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ContainerComponent from '../container';

describe('<ContainerComponent />', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <ContainerComponent><h1>TEST!</h1></ContainerComponent>,
      div,
    );
  });
  it('Matches snapshot with child', () => {
    const component = renderer.create(
      <ContainerComponent><h1>TEST!</h1></ContainerComponent>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
