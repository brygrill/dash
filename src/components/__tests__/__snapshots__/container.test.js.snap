exports[`<ContainerComponent /> Matches snapshot with child 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }>
  <div
    className="ui container"
    style={
      Object {
        "display": "flex",
        "height": "100%",
      }
    }>
    <h1>
      TEST!
    </h1>
  </div>
</div>
`;
